submissions/ses3332002/html-css-popup/build/style.css

```diff
@@ -0,0 +1,160 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: sans-serif;
+  background-color: white;
+}
+
+a {
+  text-decoration: none;
+  color: unset;
+}
+a:focus {
+  outline: 2px solid #6098f3;
+  outline-offset: 4px;
+}
+
+input[type=checkbox] {
+  position: absolute;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+header {
+  text-align: right;
+}
+
+.user_menu {
+  display: inline-flex;
+  align-items: center;
+  justify-content: space-around;
+  font-size: 0;
+  list-style: none;
+}
+
+.apps_menu {
+  display: none;
+  position: absolute;
+  top: 56px;
+  right: 40px;
+  width: 320px;
+  height: 480px;
+  border: solid 1px #6098f3;
+  box-shadow: 0px 5px 10px #6098f3;
+  text-align: center;
+  font-size: 12px;
+}
+.apps_menu::after {
+  content: "";
+  position: absolute;
+  top: -11px;
+  right: 60px;
+  width: 20px;
+  height: 20px;
+  border: none;
+  background-color: white;
+  border-top: solid 1px #6098f3;
+  border-right: solid 1px #6098f3;
+  transform: rotate(-45deg);
+  z-index: 2;
+  clip-path: polygon(-1px 0px, 21px 0px, 21px 22px);
+}
+
+.user_menu__element {
+  margin: 10px;
+  cursor: pointer;
+}
```

Comment:
It is better to write everywhere in the same style and add a empty string between the blocks. And in other such places too.

Reply:
Well, it's not me, my code is in /src/scss/style.scss.
The code that placed in /build/style.css is by sass-compiler from VS Code.